psl-core/src/main/java/org/linqs/psl/reasoner/term/online/OnlineTermStore.java

```diff
@@ -99,6 +115,92 @@ public synchronized void updateLocalVariable(ObservedAtom atom, float newValue)
         variableValues[getVariableIndex(atom)] = newValue;
     }
 
+    public synchronized boolean activateRule(Rule rule) {
+        List<Integer> rulePages = rulePageMapping.get(rule);
+        if (rulePages == null) {
+            // Rule not in model.
+            return false;
+        }
+
+        activatedRules.put(rule, true);
+
+        int activePageIndex = 0;
+        for (Integer pageIndex : rulePages) {
+            activePageIndex = activeTermPages.indexOf(pageIndex);
+            if (activePageIndex == -1) {
+                activeTermPages.add(pageIndex);
+                numPages++;
+            }
+        }
+        return true;
```

Comment:
Add a newline before the return (since it is not tightly linked with the loop).
There are a few more methods in this file that need that newline as well.